test/tests/functional/pbs_cray_suspend_resume.py

```diff
@@ -55,6 +55,7 @@ def setUp(self):
             self.skipTest("Test suite only meant to run on a Cray")
         TestFunctional.setUp(self)
 
+    @tags('cray', 'smoke')
```

Comment:
At lines 49-50 the test suite comment mentions that suspend/resume needs to be enabled on the system.  Also suspend/resume is not supported on all Cray CLE versions.  Aren't smoke tests supposed to be able to run on all platforms?  If so, I don't believe this is a good smoke test.

Reply:
I kept the tags since this is for testing a default server setting on Cray regardless of the CLE version and the enabled setting of suspend/resume.

Reply:
ok.  Makes sense.